rgbcolor.js

```diff
@@ -216,7 +216,7 @@
 			var processor = color_defs[i].process;
 			var bits = re.exec(color_string);
 			if (bits) {
-				channels = processor(bits);
+				var channels = processor(bits);

```

Comment:
One thing to keep in mind is that this is going to be function scoped, not block scoped (as a result of the `var` vs using `let`). I don't know if this has implications elsewhere.